score-lang-runtime/src/main/java/org/openscore/lang/runtime/bindings/OutputsBinding.java

```diff
@@ -49,6 +49,8 @@
                     Map<String, Serializable> scriptContext = new HashMap<>();
                     //put action outputs
                     scriptContext.putAll(actionReturnValues);
+                    //declare the new output
+                    scriptContext.put(outputKey, actionReturnValues.get(outputKey));
```

Comment:
Better to add null to the scriptContext if the action return values do not contain it